changelog/ptr-safe-end-of-deprecation.dd

```diff
@@ -12,7 +12,7 @@ The following now triggers an error instead of a deprecation:
 Use `&arr[0]` instead:
 
 ---
-@safe ubyte oops(ubyte[] arr) {
+@safe ubyte* oops(ubyte[] arr) {
```

Comment:
Huh? A pointer is returned here.
Convince yourself: https://run.dlang.io/is/2iaRFo